docs/usage/examples/opentelemetry.md

```diff
@@ -9,58 +9,79 @@ Requirements:
 Create a `docker-compose.yaml` and `otel-collector-config.yml` file as seen below in a folder.
 
 ```yaml title="docker-compose.yaml"
-version: '3'
+name: renovate-otel-demo
+
 services:
-  # Jaeger
+  # Jaeger for storing traces
   jaeger:
-    image: jaegertracing/all-in-one:1.64.0
+    image: jaegertracing/jaeger:2.1.0
     ports:
-      - '16686:16686'
-      - '4317'
+      - '16686:16686' # Web UI
+      - '4317' # OTLP GRPC
+      - '4318' # OTLP HTTP
+
+  # Prometheus for storing metrics
+  prometheus:
+    image: prom/prometheus:v3.0.1
+    ports:
+      - '9090:9090' # Web UI
+      - '4318' # OTLP HTTP
+    command:
+      - --web.enable-otlp-receiver
+      # We have to read these flags from the Dockerfile as `command` is overwriting the default flags
+      # https://github.com/prometheus/prometheus/blob/5b5fee08af4c73230b2dae35964816f7b3c29351/Dockerfile#L23-L24
+      - --config.file=/etc/prometheus/prometheus.yml
+      - --storage.tsdb.path=/prometheus
 
   otel-collector:
+    # Using the Contrib version to get access to the spanmetrics connector
+    # If you don't need the spanmetrics connector, you can use the standard version
     image: otel/opentelemetry-collector-contrib:0.116.1
-    command: ['--config=/etc/otel-collector-config.yml']
     volumes:
-      - ./otel-collector-config.yml:/etc/otel-collector-config.yml
+      - ./otel-collector-config.yml:/etc/otelcol-contrib/config.yaml
     ports:
-      - '1888:1888' # pprof extension
-      - '13133:13133' # health_check extension
-      - '55679:55679' # zpages extension
-      - '4318:4318' # OTLP HTTP
-      - '4317:4317' # OTLP GRPC
-      - '9123:9123' # Prometheus exporter
+      - '4318:4318' # OTLP HTTP ( exposed to the host )
+      - '4317:4317' # OTLP GRPC ( exposed to the host )
     depends_on:
       - jaeger
+      - prometheus
 ```
 
 ```yaml title="otel-collector-config.yml"
 receivers:
   otlp:
     protocols:
       grpc:
+        endpoint: 0.0.0.0:4317
       http:
+        endpoint: 0.0.0.0:4318
 
 exporters:
   otlp/jaeger:
     endpoint: jaeger:4317
     tls:
       insecure: true
-  logging:
-  prometheus:
-    endpoint: '0.0.0.0:9123'
+  otlphttp/prometheus:
+    endpoint: http://prometheus:9090/api/v1/otlp
+  debug:
+  # verbosity: normal
```

Comment:
The `verbosity: normal` bit is commented out. Is that intended?

Reply:
These commented out options are for users, if they run into problems. 
They would be to verbose for normal usage.